test/storage_snapper_configure_snapper_test.rb

```diff
@@ -0,0 +1,75 @@
+#!/usr/bin/env rspec
+
+ENV["Y2DIR"] = File.expand_path("../../src", __FILE__)
+
+require "yast"
+
+Yast.import "StorageSnapper"
+Yast.import "Storage"
+
+
+describe "StorageSnapper#configure_snapper?" do
+
+
+  it "configures snapper" do
+
+    data = {
+      "device" => "/dev/sda1",
+      "mount" => "/",
+      "used_fs" => :btrfs,
+      "userdata" => { "/" => "snapshots" }
+    }
+
+    Yast::Storage.stub(:GetEntryForMountpoint).with("/").once.and_return(data)
+
+    expect(Yast::StorageSnapper.configure_snapper?).to eq true
```

Comment:
NP: this will be sorter using ```subject``` http://betterspecs.org/#subject
That is, at the beginning:
```ruby
 subject { Yast::StorageSnapper.configure_snapper?) }
```
And then just
```ruby
expect(subject).to eq true
```